DevIncubator-Autopark/Vehicle.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace DevIncubatorAutopark
+{
+    internal class Vehicle : IComparable<Vehicle>
+    {
+        public readonly VehicleType VehicleType;
```

Comment:
public field it`s bad practice.
Read information about autoproperty, or replace it on private fields with get and set(if field need it) methods

Reply:
fixed